src/app/containers/ImageWithPlaceholder/index.jsx

```diff
@@ -40,23 +42,32 @@ const ImageWithPlaceholder = ({
     <Image onLoad={() => setIsLoaded(true)} {...imageProps} />
   );
 
+  const shouldPreload = !isAmp && preload;
+
   return (
-    <ImagePlaceholder css={isLoaded && `background: none`} ratio={ratio}>
-      {isAmp ? (
-        <AmpImg
-          alt={alt}
-          attribution={copyright || ''}
-          layout="responsive"
-          src={src}
-          srcset={srcset}
-          height={height}
-          width={width}
-        />
-      ) : (
-        renderImage(imageToRender, lazyLoad, fallback)
+    <>
+      {shouldPreload && (
+        <Helmet>
+          <link rel="preload" as="image" href={src} imagesrcset={srcset} />
+        </Helmet>
       )}
-      {children}
-    </ImagePlaceholder>
+      <ImagePlaceholder css={isLoaded && `background: none`} ratio={ratio}>
+        {isAmp ? (
+          <AmpImg
```

Comment:
Also it looks like amp does support preloading hero images: https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/optimize_amp/#preload-hero-images.

Probably be worth having this as a separate issue though.